src/Classes/MenuItemText.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace OptimistDigital\MenuBuilder\Classes;
+
+abstract class MenuItemText
```

Comment:
Can't you extend the MenuLinkable class instead of creating a new abstract one?

Reply:
It seems wrong to extend MenuLinkable since it's not linkable, should I make a base class MenuItemBase.php that all the types extend?